### PR TITLE
users/telemetry: add device telemetry page

### PR DESCRIPTION
**Pull request by yaarith**
Signed-off-by: Yaarit Hatuka <yaarit@redhat.com>

### CODE REVIEW

File: src/en/users/telemetry/device-telemetry/index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        </p>
+        <p class="p">
+          Each file contains the data reported during a single month, according to the
+          file’s name (e.g. device_health_metrics_2020-01.zip contains all device health
+          reports of January 2020).
+        </p>
+        <p class="p">
+          Please note: the raw data contains reports from devices that could not scrape
+          their health metrics due to software issues (e.g. an older smartmontools
+          version that does not support the JSON output option), this is when the
```
Comment (liewegas) [inline src/en/users/telemetry/device-telemetry/index.html:64]: ``;`` instead of ``,``

File: src/en/users/telemetry/device-telemetry/index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      </p>
+      <p class="p">
+        The data collected is anonymized on the client side before it is sent
+        to the telemetry server. Ceph does not collect any identifying
+        information, and replaces both host name and drive serial number with
+        random generated UUIDs.
+      </p>
+        <h2 class="h2">Data format</h2>
+        <p class="p">
+          The data is split to zipped CSV files, which have 4 columns:
```
Comment (liewegas) [inline src/en/users/telemetry/device-telemetry/index.html:48]: s/to/into/

File: src/en/users/telemetry/device-telemetry/index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+          Each file contains the data reported during a single month, according to the
+          file’s name (e.g. device_health_metrics_2020-01.zip contains all device health
+          reports of January 2020).
+        </p>
+        <p class="p">
+          Please note: the raw data contains reports from devices that could not scrape
+          their health metrics due to software issues (e.g. an older smartmontools
+          version that does not support the JSON output option), this is when the
+          ‘invalid’ column is set to ‘true’. This does not reflect any problem with the
+          device’s health, it simply means that the health metrics could not be retrieved
```
Comment (liewegas) [inline src/en/users/telemetry/device-telemetry/index.html:66]: s/,/;/

File: src/en/users/telemetry/device-telemetry/index.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+          due to software issues on that host. We chose to include these invalid
+          telemetry reports since they do indicate the device is still active, which can
+          contribute to certain model training approaches.
+        </p>
+        <p class="p">
+          The drive telemetry data does not contain actual drive’s state labels. We do
```
Comment (liewegas) [inline src/en/users/telemetry/device-telemetry/index.html:72]: s/drive's/drives'/.  Maybe clarify what 'state' means?  e.g., ``...state (i.e., healthy vs failed)``

**Review by liewegas — APPROVED** [review]
few grammar nits, but otherwise looks great!

**Comment by yaarith** [comment]
thanks, fixed!
should I squash this second commit?

**Pull request merged**